model/historical_test_data.go

```diff
@@ -0,0 +1,69 @@
+package model
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/cedar"
+	"github.com/pkg/errors"
+)
+
+// HistoricalTestData describes aggregated test result data for a given date
+// range.
+type HistoricalTestData struct {
+	Info            HistoricalTestDataInfo
+	NumPass         int       `bson:"num_pass"`
+	NumFail         int       `bson:"num_fail"`
+	AverageDuration float64   `bson:"average_duration"`
```

Comment:
do we want to store total time in some units? also units are unclear.

Reply:
Would you rather store totalTime and calculate average duration on the fly using NumPass + NumFail?

Reply:
Just to clarify, this is going to be stored in s3? If that's true (and it probably ought to be?) then we should probably store everything (components and average?)

Reply:
Don't think there is anything else to store here? Average will be calculated as test results come in, don't see a compelling reason to store each individual test duration.

Reply:
Unless we are planning to do more complex rollups?